docs/model-dev-guide/dtrain/dtrain-introduction.rst

```diff
@@ -8,93 +8,93 @@
-consists of a set of workers, where each worker is assigned to a unique compute accelerator such as
-a GPU or a TPU. Each worker maintains a copy of the model parameters (weights that are being
-trained), which is synchronized across all the workers at the start of training.
+Determined employs data parallelism in its approach to distributed training. Data parallelism for
+deep learning consists of a set of workers, where each worker is assigned to a unique compute
+accelerator such as a GPU or a TPU. Each worker maintains a copy of the model parameters (weights
+that are being trained), which is synchronized across all the workers at the start of training.
 
 After initialization is completed, distributed training in Determined follows a loop where:
 
 #. Every worker performs a forward and backward pass on a unique mini-batch of data.
-#. As the result of the backward pass, every worker generates a set of updates to the model
-   parameters based on the data it processed.
-#. The workers communicate their updates to each other, so that all the workers see all the updates
+#. As a result of the backward pass, every worker generates a set of updates to the model parameters
+   based on the data it processed.
+#. The workers communicate their updates to each other so that all the workers see all the updates
    made during that batch.
 #. Every worker averages the updates by the number of workers.
 #. Every worker applies the updates to its copy of the model parameters, resulting in all the
    workers having identical solution states.
 #. Return to the first step.
 
-***********************************************
- Reduce Computation and Communication Overhead
-***********************************************
+*************************************************
+ Reducing Computation and Communication Overhead
+*************************************************
 
-Of the steps involved in the distributed training loop, step 1 and step 2 introduce the majority of
-the computational overhead. To reduce computational overhead, it's recommended that users maximize
-the utilization of their GPU. This is typically done by using the largest batch size that fits into
-memory. When performing distributed training, to reduce the computational overhead it's recommended
-to set the ``global_batch_size`` to the largest batch size that fits into a single GPU multiplied
-times the number of slots. This is commonly referred to as *weak scaling*.
+In the distributed training loop, the first two steps bring about the most computational burden,
+known as computational overhead. To reduce this computational overhead, we recommend using your GPU
+at its maximimum capacity. This is typically achieved by using the largest batch size that fits into
+memory. More specifically, we recommend setting the ``global_batch_size`` to the largest batch size
+that fits into a single GPU multiplied by the number of slots. This is commonly referred to as *weak
```

Comment:
nit: I would remove the _weak scaling_ sentence because it's both overly-jargony (imo), but also not accurate in this paragraph.

Reply:
@garrett361 let's revisit this - see the other comment about weak scaling